LEGO1/act2policestation.cpp

```diff
@@ -0,0 +1,9 @@
+#include "act2policestation.h"
+
+// OFFSET: LEGO1 0x100d53ac
```

Comment:
Incorrect address: this is the address of the vtable entry, but not the address of the actual function